### DISCUSSION TITLE
IAM Role usage with Lambda Plugin

### DISCUSSION THREAD

**Discussion by spencer-steed** [RoadieHQ/roadie-backstage-plugins#619] [Q&A]
https://github.com/RoadieHQ/roadie-backstage-plugins/issues/179
Is this feature available with any kind of workaround currently? We have lambdas that live in seperate accounts from where backstage is hosted, so something like this would be very helpful.

**Comment by Xantier** [comment]
At the moment there is no workaround before the work in #179 would be implemented. Would that be something that you could possibly contribute to the project? We don't at the moment have internal need for this functionality but I can see it being useful.
